ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template1.kt

```diff
@@ -163,7 +166,11 @@ private fun CircleMask(content: @Composable () -> Unit) {
                 scaleX = Template1UIConstants.circleScale,
                 scaleY = Template1UIConstants.circleScale,
             )
-            .clip(shape = CircleShape),
+            .clip(
+                shape = GenericShape { size, _ ->
+                    addOval(Rect(Offset.Zero, size))
+                },
```

Comment:
`CircleShape` is defined like this: 
```kotlin
RoundedCornerShape(50)
```

Which explained why the clip shape was totally off when transforming it.